policyengine_us/variables/gov/states/ct/tax/income/subtractions/ct_social_security_benefit_adjustment.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class ct_social_security_benefit_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Connecticut social security benefit subtraction"
+    definition_period = YEAR
+    defined_for = "ct_social_security_benefit_subtraction_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.social_security.taxability
+        income_base_amount = p.threshold.lower
+        ss_agi_frac = p.rate.lower
+        filing_status = tax_unit("filing_status", period)
+        base_amount = income_base_amount[filing_status]
+        ss_rate = parameters(
+            period
+        ).gov.states.ct.tax.income.subtractions.social_security.rate
+        # is_eligible = tax_unit(
+        #     "ct_social_security_benefit_adjustment_eligible", period
+        # )
+        ss_benefit = add(tax_unit, period, ["social_security"])
```

Comment:
if this following passage is the right one then it should be `taxable_social_security` 

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/5b67287d-6ea5-4921-9431-9b5db213df32)